lib/tasks/test_replication.rake

```diff
@@ -75,6 +75,10 @@ class EvmTestSetupReplication
     `git branch -D #{TEST_BRANCH}`
   end
 
+  def fetch_command
+    "git fetch #{"--depth=1 " if ENV["CI"]}http://github.com/ManageIQ/manageiq.git refs/heads/euwe:#{TEST_BRANCH}"
```

Comment:
My mind wants to think this is invalid syntax because the `"` isn't escaped within the string interpolation.    It's easier for my mind to read this as:

```ruby
"git fetch #{'--depth=1 ' if ENV['CI']}http://github.com/ManageIQ/manageiq.git refs/heads/euwe:#{TEST_BRANCH}"
```